README.md

```diff
@@ -434,6 +436,10 @@ As part of changes being introduced in version 2.6.0 we are deprecating and remo
 
 [See Tree-Shaking Recommendations](TreeShakingRecommendations.md)
 
+## Service Notification
+
+As part of changes being introduced in version 3.0.3, we are intergrating [cfgSync plugin](https://github.com/microsoft/ApplicationInsights-JS/tree/main/extensions/applicationinsights-cfgsync-js) and [throttle manager](./docs/ThrottleMgr.md) to AISKU. By default, these components will be disabled unless you choose to opt in experimental features via the `featureOptIn` of your root config.
```

Comment:
Probably should rephrase the "By default, these components will be disabled" to something like

These components are currently disabled but will be enabled during the current initial release, but will be enabled by default in future releases after extensive production level testing is complete.